docker-compose.yml

```diff
@@ -70,3 +71,14 @@ services:
     networks:
       - dev
       - test
+    ports:
+      - 5432:5432
+
+  gcs:
+    image: fsouza/fake-gcs-server
+    command: ["-scheme", "http", "-port", "4443", "-external-url", "http://localhost:4443", "-backend", "memory"]
```

Comment:
Does this work with localhost? I would guess it has to be set to '0.0.0.0' 😮

Reply:
works :D but changed it to 0.0.0.0